channels/pe.m3u

```diff
@@ -93,3 +95,15 @@ https://live.siete.us/uraniotv/uraniotv/index.m3u8
 https://rtmp02.portalexpress.es/primaveratv/primaveratv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/tvnorte7/picture?width=320&height=320" group-title="Local",TV Norte (Chiclayo)
 https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/TVNORTEHD
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/PlanetaTeleInformativo/picture?width=320&height=320" group-title="Local",Planeta TV (Moyobamba)
+https://media2.cdnlayer.biz:8081/8018/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/telesurexpresion/picture?width=320&height=320" group-title="Local",Telesur (Ilo)
+https://qlobbidev.s.llnwi.net/telesur3/hls/ilo.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwnhI0HKESXrVQ8-ktPCeRap6-cP_4upPtEpAlwPa=s88-c-k-c0x00ffffff-no-rj" group-title="Music",Hatun TV
+https://live.siete.us:19360/bestcablehatuntv/bestcablehatuntv.m3u8
```

Comment:
Link Not Working - All the rest are fine.

Reply:
Fixed